syncer/db_test.go

```diff
@@ -32,3 +33,34 @@ func (s *testSyncerSuite) TestGetServerID(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(id, Greater, int64(0))
 }
+
+func (s *testSyncerSuite) TestBinaryLogs(c *C) {
+	files, err := getBinaryLogs(s.db)
+	c.Assert(err, IsNil)
+	c.Assert(files, Not(HasLen), 0)
+
+	pos := mysql.Position{
+		Name: files[len(files)-1].name,
+		Pos:  0,
+	}
+
+	remainingSize, err := countBinaryLogsSize(pos, s.db)
+	c.Assert(err, IsNil)
+	c.Assert(remainingSize, Equals, files[len(files)-1].size)
+	fileNum := len(files)
```

Comment:
why not putting it before L42 to save some code

Reply:
done https://github.com/pingcap/dm/pull/150/commits/4e689c7a16e3a3118df05cb11ded5ac48c0a3935